package.json

```diff
@@ -188,7 +188,7 @@
     "vinyl-fs": "4.0.0",
     "vite": "~4.4.0",
     "vite-tsconfig-paths": "4.2.0",
-    "vitest": "0.32.4",
+    "vitest": "0.34.4",
     "vue-tsc": "1.8.10",
     "webpack": "5.88.2",
     "webpack-virtual-modules": "0.5.0"
```

Comment:
The version of 'vitest' has been upgraded. Make sure to test the new version thoroughly to ensure it doesn't break existing functionality.